frontend/src/components/mine/Compliance/MineComplianceInfo.js

```diff
@@ -24,90 +25,187 @@ const defaultProps = {
+                        <p className="info-display-label">WARNINGS ISSUED IN THE PAST YEAR</p>
+                      </div>
+                    </Col>
+                    <Col span={4} />
+                    <Col span={4}>
+                      <div className="center">
+                        <p className="info-display">{this.props.mineComplianceInfo.advisories}</p>
+                        <p className="info-display-label">ADVISORIES ISSUED IN THE PAST YEAR</p>
+                      </div>
+                    </Col>
+                    <Col span={4} />
+                    <Col span={4}>
+                      <div className="center">
+                        <p className="info-display">
+                          {this.props.mineComplianceInfo.section_35_orders} - Section 35
+                        </p>
+                        <p className="info-display-label">TOTAL</p>
+                      </div>
+                    </Col>
+                    <Col span={2} />
+                  </Row>
+                </div>
+              )}
+              {!this.props.mineComplianceInfo && <NullScreen type="generic" />}
+            </div>
+          )}
+        </div>
+        <br />
+        <br />
+        <div>
+          {this.props.mineComplianceInfo.open_orders && (
+            <div>
+              <h4>Open Orders</h4>
+              <br />
+              <Row gutter={16} justify="center" align="top">
+                <Col span={2} />
+                <Col span={4}>
+                  <h5>Order #</h5>
+                </Col>
+                <Col span={4}>
+                  <h5>Violation</h5>
+                </Col>
+                <Col span={4}>
+                  <h5>Report #</h5>
+                </Col>
+                <Col span={4}>
+                  <h5>Inspector Name</h5>
+                </Col>
+                <Col span={4}>
+                  <h5>Due Date</h5>
+                </Col>
+                <Col span={2} />
+              </Row>
+              <Divider type="horizontal" className="thick-divider" />
+              {this.props.mineComplianceInfo.open_orders
+                .sort((order1, order2) => {
+                  if (!(Date.parse(order1.due_date) === Date.parse(order2.due_date)))
+                    return Date.parse(order1.due_date) > Date.parse(order2.due_date) ? 1 : -1;
+                  return order1.order_no > order2.order_no ? 1 : -1;
+                })
+                .slice(this.state.minOrderList, this.state.maxOrderList)
+                .map((order, id) => {
+                  return (
+                    <div key={order.order_no}>
+                      <Row gutter={16} justify="center" align="top">
+                        <Col span={2}>
+                          {order.overdue && order.due_date != null ? (
+                            <img style={{ padding: "5px" }} src={RED_CLOCK} alt="Edit TSF Report" />
```

Comment:
There was CSS on other `img` tags to accomplish the same task, it wasn't a reusable class though.

This could be a good time to make a reusable class, as I believe most images are going to have this spacing issue.

Reply:
There were about 5 or so instances of this so I added a class to layout called padding-small which is just 5px padding.